config.json

```diff
@@ -2096,6 +2280,14 @@
         "dependencyOnly": false,
         "frozen": true
       },
+      {
+        "groupId": "com.google.ads.interactivemedia.v3",
+        "artifactId": "interactivemedia",
```

Comment:
This package is licensed as Android SDK license (https://maven.google.com/web/index.html?q=media3#com.google.ads.interactivemedia.v3:interactivemedia:3.32.0), it needs to be bound in GPS repo so we do not ship the proprietary code in our package.

Reply:
moved